src/language-css/parser-postcss.js

```diff
@@ -32,6 +32,17 @@ function parseValueNodes(nodes) {
   for (let i = 0; i < nodes.length; ++i) {
     const node = nodes[i];
 
+    if (node.type === "func" && node.value === "selector") {
+      node.group.groups = [
+        parseSelector(
+          rootValueText.slice(
+            rootValueText.indexOf("selector(") + "selector(".length,
+            -1
+          )
```

Comment:
I think https://github.com/prettier/prettier/pull/8545#issuecomment-645276974 can be fixed if we cut the string from `groups first param start` to `groups last param end`.